lib/text/ui_text_displayer.js

```diff
@@ -147,6 +147,7 @@ shaka.text.UITextDisplayer = class {
   configure(config) {
     this.config_ = config;
     this.configureCaptionsTimer_();
+    this.updateCaptions_(/* forceUpdate= */ true);
```

Comment:
If we're only updating the caption for the sake of this new font size config, we should probably check if the config value has actually changed before we force-update the captions.

Reply:
We can't do that because we're not making a copy... it's a reference

Reply:
The fontScaleFactor config value is just a number, no? So I don't see why we can't do something like:
```js
    const oldFontScaleFactor = this.config_ ? this.config_.fontScaleFactor : 1;
    this.config_ = config;
    this.configureCaptionsTimer_();
    const newFontScaleFactor = this.config_ ? this.config_.fontScaleFactor : 1;
    if (newFontScaleFactor != oldFontScaleFactor) {
      this.updateCaptions_(/* forceUpdate= */ true);
    }
```

Reply:
I tested that and It doesn't work

Reply:
The config obj is altered from upper logic, but it remains the same reference. My theory is that configure isn't getting called a lot anyways, so this will rarely be executed.